setup.cfg

```diff
@@ -0,0 +1,66 @@
+[metadata]
+name = hankel
+author = Steven Murray
+author_email = steven.murray@curtin.edu.au
+license = MIT
+description = Hankel Transformations using method of Ogata 2005
+url = https://github.com/steven-murray/hankel
+long_description = file: README.rst
+long_description_content_type = text/x-rst
+classifiers =
+    Development Status :: 5 - Production/Stable
+    Intended Audience :: Developers
+    Intended Audience :: End Users/Desktop
+    Intended Audience :: Science/Research
+    License :: OSI Approved :: MIT License
+    Natural Language :: English
+    Operating System :: Unix
+    Programming Language :: Python
+    Programming Language :: Python :: 3
+    Programming Language :: Python :: 3.5
+    Programming Language :: Python :: 3.6
+    Programming Language :: Python :: 3.7
+    Programming Language :: Python :: 3.8
+    Programming Language :: Python :: 3 :: Only
+    Topic :: Scientific/Engineering
+    Topic :: Utilities
+
+project_urls =
+    Documentation = https://hankel.readthedocs.io
+    Source = https://github.com/steven-murray/hankel
+    Tracker = https://github.com/steven-murray/hankel/issues
+    Changelog = https://github.com/steven-murray/hankel/blob/master/CHANGELOG.rst
+    Conda-Forge = https://anaconda.org/conda-forge/hankel
+
+[options]
+packages = hankel
+python_requires = >=3.5
+setup_requires = setuptools_scm
+install_requires =
+    numpy>=1.9.3
+    scipy>=0.16.1
+    mpmath>=1.0.0
+
+[options.extras_require]
+dev =
+    pre-commit
+    pytest
+    pytest-cov
+    ipykernel
+    nbsphinx
+    numpydoc
+    pandoc
+    sphinx>=2.0<3
+    sphinx_rtd_theme
+
+tests = pre-commit
+    ; pytest
```

Comment:
why is this commented out? We do need `pytest` for tests (not pytest-cov though)

Reply:
it is not a comment. It just the conversion tool has worked incorrectly. Fixed